client.go

```diff
@@ -6,33 +6,38 @@ import (
 )
 
 const apiEndpoint = "https://overpass-api.de/api/interpreter"
+const apiOutputFormatXml = "xml"
+const apiOutputFormatJson = "json"
 
 type HTTPClient interface {
 	PostForm(url string, data url.Values) (*http.Response, error)
 }
 
 // A Client manages communication with the Overpass API.
 type Client struct {
-	apiEndpoint string
-	httpClient  HTTPClient
-	semaphore   chan struct{}
+	apiEndpoint     string
+	apiOutputFormat string
+	httpClient      HTTPClient
+	semaphore       chan struct{}
 }
 
 // New returns Client instance with default overpass-api.de endpoint.
 func New() Client {
-	return NewWithSettings(apiEndpoint, 1, http.DefaultClient)
+	return NewWithSettings(apiEndpoint, apiOutputFormatXml, 1, http.DefaultClient)
```

Comment:
Why not add this format at the query level, so we can use the same client while querying multiple times with different formats ?

Reply:
Updated the code to get the output format by parsing the query string. It looks much better now. The only difference is that go-overpass now expects XML format by default.